openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/weighting/HeatStressWeighting.java

```diff
@@ -29,9 +29,11 @@ public HeatStressWeighting(FlagEncoder encoder, PMap map, GraphStorage graphStor
 
     @Override
     public double calcWeight(EdgeIteratorState edgeState, boolean reverse, int prevOrNextEdgeId) {
+        // TODO: this is a short term solution to be adapted when
+        //       AdditionWeighting will be replaced
         if (heatStressStorage != null) {
             int stressLevel = heatStressStorage.getEdgeValue(EdgeIteratorStateHelper.getOriginalEdge(edgeState), columnIndex, buffer);
-            return stressLevel * weightingFactor / 100;
+            return stressLevel * weightingFactor / 100 + 1.0;
         }
 
         return 1.0;
```

Comment:
Does the default return value have to be adated if we change the value in line 36?

Reply:
The default return value meaning 'no change' should be 1.0 in case of AdditionWeighting and must be adapted when a more sensible model will be implemented.